engine/runtime/src/main/java/org/enso/interpreter/runtime/data/EnsoMultiValue.java

```diff
@@ -432,9 +427,17 @@ public abstract static class CastToNode extends Node {
      *
      * @param type the requested t
      * @param mv a multi value
+     * @param reorderOnly allow (modified) {@link EnsoMultiValue} to be returned
+     * @param allTypes should we search all types or nu
```

Comment:
Half finished sentence in the doc, please fix it.

Reply:
Done in 9929fe8d26